### PR TITLE
PXB-3196 Release notes for Percona XtraBackup 8.0.35-30

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/release-notes.md
	new file:   docs/release-notes/8.0/8.0.35-30.0.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0/8.0.35-30.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Bug fix
+
+* [PXB-3003](https://jira.percona.com/browse/PXB-3003): The `log_files_find_and_analyze` option had a potential race condition when the user listed the available files and tried to open them. In between these two operations, the server could delete the log files, causing the `log_files_find_and_analyze` option to return an error.
+
+* [PXB-3139](https://jira.percona.com/browse/PXB-3139): Fix for gcc-13 and clang-16 compilation issues.
+
+* [PXB-3147](https://jira.percona.com/browse/PXB-3147): Percona XtraBackup failed to execute the `DO innodb_redo_log_consumer_register("PXB");` query.
+
+* [PXB-3181](https://jira.percona.com/browse/PXB-3181): The use of the `innodb-use-native-aio=true` option resulted in printing duplicate timestamps.
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.35-30.0.md]: I think there is one more bug fix from marce. @altmannmarcelo ? the partial download fix for xbcloud

Comment (altmannmarcelo) [reply docs/release-notes/8.0/8.0.35-30.0.md]: Please add 
https://jira.percona.com/browse/PXB-3173
https://jira.percona.com/browse/PXB-3141

File: docs/release-notes/8.0/8.0.35-30.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Release highlights
+
+This release merges the MySQL 8.0.35 code base.
+
+## Bug fix
+
+* [PXB-3003](https://jira.percona.com/browse/PXB-3003): The `log_files_find_and_analyze` option had a potential race condition when the user listed the available files and tried to open them. In between these two operations, the server could delete the log files, causing the `log_files_find_and_analyze` option to return an error.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.35-30.0.md]: This is actually nothing to do with user options

Comment (altmannmarcelo) [reply docs/release-notes/8.0/8.0.35-30.0.md]: Add something like:

A possible race conditing while PXB is discovering redo logs to parse and server purging them.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0/8.0.35-30.0.md (hunk 2 above)
Comment (patrickbirch) [reply docs/release-notes/8.0/8.0.35-30.0.md]: Percona XtraBackup discovering redo logs to parse and the server purging redo logs at the same time could cause a race condition.

File: docs/release-notes/8.0/8.0.35-30.0.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.35-30.0.md]: added

File: docs/release-notes/8.0/8.0.35-30.0.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.35-30.0.md]: fixed

**Review by satya-bodapati — APPROVED** [review]
LGTM! @altmannmarcelo please verify if all fixes are documented :)

**Review by patrickbirch — APPROVED** [review]
Good

File: docs/release-notes/8.0/8.0.35-30.0.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+* [PXB-3139](https://jira.percona.com/browse/PXB-3139): Fix for gcc-13 and clang-16 compilation issues.
+
+* [PXB-3147](https://jira.percona.com/browse/PXB-3147): Percona XtraBackup failed to execute the `DO innodb_redo_log_consumer_register("PXB");` query.
+
+* [PXB-3181](https://jira.percona.com/browse/PXB-3181): The use of the `innodb-use-native-aio=true` option resulted in printing duplicate timestamps.
+
+* [PXB-3173](https://jira.percona.com/browse/PXB-3173): The xbcloud was refactored to use multiple threads with a global list of files. This list did not allow users to download a subset of files.
+
+* [PXB-3141](https://jira.percona.com/browse/PXB-3141): Adjusted folder creation on `ds_fifo.cc` and xbcloud to use 777 as permission.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.35-30.0.md]: 777 here can be misinterpreted as too open, which would be true for a file, but not for a folder. I would adjust this to:

Adjusted folder creation on `ds_fifo.cc` and xbcloud to use proper permission.

File: docs/release-notes/8.0/8.0.35-30.0.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+* [PXB-3168](https://jira.percona.com/browse/PXB-3168): When the write load was high, a backup failed with a `log block numbers mismatch` error.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.35-30.0.md]:
```suggestion
* [PXB-3168](https://jira.percona.com/browse/PXB-3168): Under high write load, a backup might failed with a `log block numbers mismatch` error.
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0/8.0.35-30.0.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.35-30.0.md]: fixed

File: docs/release-notes/8.0/8.0.35-30.0.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.35-30.0.md]: fixed

**Pull request merged**